PocketKit/Sources/Textile/Views/ItemRow.swift

```diff
@@ -71,6 +81,17 @@ public struct ItemRowView<Model: ItemRow>: View {
     }
 }
 
+extension ItemRowView {
+    private var textAlignment: TextAlignment {
+        switch model.languageDirection {
+        case .leftToRight:
+            return layoutDirection == .leftToRight ? .leading : .trailing
+        case .rightToLeft:
+            return layoutDirection == .rightToLeft ? .leading : .trailing
+        }
+    }
+}
```

Comment:
Interesting. As I understand it, leading and trailing will be be left/right or right/left, depending on the phone's locale. So, if I'm in an Arabic locale, leading will be right edge, and trailing the left. Will this properly align English text in that locale, or have the opposite intended effect?

Reply:
@nzeltzer I think screenshots will showcase this best:

System language: left-to-right, with an example of both left-to-right and right-to-left article titles
![image](https://user-images.githubusercontent.com/1158092/125484771-70c56b0e-4b19-4474-8e37-8ad2df8bb3d0.png)

System language: right-to-left with an example of both left-to-right and right-to-left article titles
![image](https://user-images.githubusercontent.com/1158092/125484866-0dde17ed-d9ae-4943-8d09-95d711010904.png)